BEST_PRACTICES.md

```diff
@@ -334,50 +304,51 @@ const request = {
 
 client
   .execute(request)
-  .then(result => {})
-  .catch(error => {})
+  .then((result) => {})
+  .catch((error) => {})
 ```
 
 ## Using HTTP client
 
-The JS/TS SDK uses [fetch API](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API) and its close relatives like [isomorphic-fetch](https://github.com/matthew-andrews/isomorphic-fetch), [whatwg-fetch](https://github.com/whatwg/fetch) or [unfetch](https://github.com/developit/unfetch).
+The Commercetools JS/TS SDK uses [fetch API](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API) and its close relatives like [isomorphic-fetch](https://github.com/matthew-andrews/isomorphic-fetch), [whatwg-fetch](https://github.com/whatwg/fetch) or [unfetch](https://github.com/developit/unfetch).
 It is important to also note that due to changes in version 3 of `node-fetch`, it is expected that all projects using the v3 of `node-fetch` within the SDK must be in ESM (.mjs) module system. The JS/TS SDK fully supports v2 of `node-fetch` without any restrictions.
 
 ## The `Process` Function
 
-The JS/TS SDK exposes a `Process` function that can be called to process batch requests. This function takes a request parameter, a callback that will be called on each batch request and an option that is an object with key `total` and a boolean `accumulate`.
+The Commercetools JS/TS SDK exposes a `Process` function that can be called to process batch requests. This function takes a request parameter, a callback that will be called on each batch request and an option that is an object with key `total` and a boolean `accumulate`.
 
 ```ts
-import { Process, ClientBuilder } from '@commercetools/sdk-client-v2'
+import { Process, ClientBuilder } from '@commercetools/ts-client'
 import { createApiBuilderFromCtpClient } from '@commercetools/platform-sdk'
 
-...
-const apiRoot = createApiBuilderFromCtpClient(
-  new ClientBuilder()
-    .withProjectKey(projectKey)
-    .withClientCredentialsFlow(authMiddlewareOptions)
-    .withHttpMiddleware(httpMiddlewareOptions)
-    .build()
-)
+//...
```

Comment:
I believe this shouldn't be a comment, it's a redaction that shows some code were excluded for brevity. 
```suggestion
...
```

Reply:
I believe that just this it's better to be commented otherwise can be confused with Spread operator or Rest parameters.